src/main/scala/is/hail/rvd/UnpartitionedRVD.scala

```diff
@@ -91,5 +91,13 @@ class UnpartitionedRVD(val rowType: TStruct, val crdd: ContextRDD[RVDContext, Re
     OrderedRVD.shuffle(ordType, newPartitioner, filtered)
   }
 
+  def subsetPartitions(keep: Array[Int]): OrderedRVD = {
+    require(keep.length <= crdd.partitions.length, "tried to subset to more partitions than exist")
+    require(keep.isIncreasing && (keep.isEmpty || (keep.head >= 0 && keep.last < crdd.partitions.length)),
```

Comment:
I think it makes more sense to only have the second require here and where you copied this from, with the word "sorted" replaced by "increasing".

Reply:
I left the first assert (cheap, good error message) but changed sorted to increasing.